javascript-frameworks-resources.md

```diff
@@ -83,15 +83,15 @@
 
 * [Knockout.js Succinctly](https://www.syncfusion.com/resources/techportal/ebooks/knockoutjs) (PDF, Kindle) *(Just fill the fields with any values)*
 * tech.pro: Knockout.js tutorial video series
-  * [Lesson 1 - Introduction](http://tech.pro/tutorial/1562/knockoutjs-lesson-1-introduction)
-  * [Lesson 2 - Initialising the application](http://tech.pro/tutorial/1563/knockoutjs-lesson-2-initialising-the-application)
-  * [Lesson 3 - Adding a viewModel](http://tech.pro/tutorial/1564/knockoutjs-lesson-3-adding-a-viewmodel)
-  * [Lesson 4 - Basic bindings](http://tech.pro/tutorial/1565/knockoutjs-lesson-4-basic-bindings)
-  * [Lesson 5 - Observable arrays](http://tech.pro/tutorial/1566/knockoutjs-lesson-5-observable-arrays)
-  * [Lesson 6 - The foreach binding](http://tech.pro/tutorial/1567/knockoutjs-lesson-6-the-foreach-binding)
-  * [Lesson 7 - The event binding](http://tech.pro/tutorial/1568/knockoutjs-lesson-7-the-event-binding)
-  * [Lesson 8 - The click binding](http://tech.pro/tutorial/1569/knockoutjs-lesson-8-the-click-binding)
-  * [Lesson 9 - Custom bindings](http://tech.pro/tutorial/1570/knockoutjs-lesson-9-custom-bindings)
+  * [Lesson  1 - Introduction](http://tech.pro/tutorial/1562/knockoutjs-lesson-1-introduction)
```

Comment:
As you probably noticed, I wrote a linter to check the style guidelines on every PR and commits.

I added an alphabetization rule to the linter, and alphabetized all entries according to this new rule. Since it's ordering based on text, `10` comes before `2`. I added these (mostly invisible) spaces to make the linter stop complaining.